.github/workflows/Github_pytest.yml

```diff
@@ -101,11 +98,11 @@ jobs:
         run: |
           python -m pytest ndarrays/ -rx
         working-directory: ./tests
-      #- name: Parallel tests
-      #  run: |
-      #    mpiexec -n 4 python -m pytest epyccel/test_parallel_epyccel.py -v -m parallel -rx
-      #    mpiexec -n 4 python -m pytest epyccel -v -m parallel -rx
-      #  working-directory: ./tests
+      - name: Parallel tests
+        run: |
+          mpiexec -n 4 python -m pytest epyccel/test_parallel_epyccel.py -v -m parallel -rx
+          # mpiexec -n 4 python -m pytest epyccel -v -m parallel -rx
+        working-directory: ./tests
```

Comment:
Instead of separating serial and parallel tests in a manual fashion, could we not use `./.github/actions/pytest_run`?

Reply:
No. The pytest action uses bash so we can't use it for Windows

Reply:
I see. Can you please add a comment to issue #517 about this?

Reply:
done